hw/top_earlgrey/clkgen_xil7series.core

```diff
@@ -0,0 +1,20 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# This is the clock generation module for FPGA targets.
```

Comment:
You can remove this comment, it's duplicated in the description below.

Reply:
ack